### PR TITLE
[exobundle] Add detail for score in the QCM

### PR DESCRIPTION
**Pull request by charlotte42**
Begin #238

### CODE REVIEW

File: plugin/exo/Transfer/Json/QuestionHandler/QcmHandler.php [hunk 1]
```diff
@@ -178,6 +187,15 @@ public function convertInteractionDetails(Question $question, \stdClass $exportD
 
         $exportData->scoreTotal = $this->container->get('ujm.exo.qcm_service')->maxScore($interaction);
 
+        if(!$interaction->getWeightResponse()){
+            $exportData->score=array('type'=>'fixed',
+                'success' =>$interaction->getScoreRightResponse(),
+                'failure' =>$interaction->getScoreFalseResponse());
+        }
+        else{
+            $exportData->score=array('type'=>'sum');
```
Comment (Elorfin) [inline plugin/exo/Transfer/Json/QuestionHandler/QcmHandler.php]: Coding style : 
- Use array shorthand syntax `[]`
- Each item of the array goes on a new line with one more indent.
- PHP CS Fixer will fix the missing whitespace around operands.

Comment (charlotte42) [inline plugin/exo/Transfer/Json/QuestionHandler/QcmHandler.php]: For import method, I have followed the tests. Do you want that I delete this method ?

**Comment by stefk** [comment]
LGTM

**Comment by Elorfin** [comment]
LGTM

**Pull request merged**